sale_order_type/models/sale_order_type.py

```diff
@@ -42,32 +31,24 @@ def default_picking_policy(self):
     warehouse_id = fields.Many2one(
         comodel_name="stock.warehouse", string="Warehouse", check_company=True
     )
-    picking_policy = fields.Selection(
-        selection="_get_selection_picking_policy",
-        string="Shipping Policy",
-        default=default_picking_policy,
-    )
     company_id = fields.Many2one(
         comodel_name="res.company", default=lambda self: self.env.company, store=True,
     )
     payment_term_id = fields.Many2one(
-        comodel_name="account.payment.term", string="Payment Term", check_company=True
+        comodel_name="account.payment.term",
+        string="Payment Term",
+        sale_order_field="payment_term_id",
```

Comment:
I find it weird that you're removing the `check_company=True` across fields like this one.

I guess it was doing nothing because this model doesn't have a `company_id` field, but in that case it seems more sensible to add that field, don't you think?

Reply:
I find that weird too, don't remember my reasoning back then if there was any, will re-add